cmd/monaco/delete/delete.go

```diff
@@ -38,6 +38,9 @@ import (
 func Delete(ctx context.Context, environments manifest.Environments, entriesToDelete delete.DeleteEntries) error {
 	var envsWithDeleteErrs []string
 	for _, env := range environments {
+		if env.Skip {
```

Comment:
We now have a lot of these `env.Skip` checks (7), and I am thinking if it would be possible to not even pass them into the consuming functions, aka filter them beforehand.

Reply:
I was wondering the same. Would it be possible to add to the `Manifest` struct something like a getter method where only active Environments are returned?

Reply:
This is a fair point - let's do this!

I just tried with an iterator, but it doesn't really solve the problem: defining it on the `Manifest` would require changing many function signatures (including this one), definition it on `manifest.Environments` doesn't improve the problem, as you could still forget to use it. So I think I'll go for an actual getter on the `Manifest` that does the filtering when called.